lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -97,7 +97,36 @@ define((require, exports, module) ->
                 result.mismatch = false
         return result
 
+    higlightInitBind = (editor) ->
+        session = editor.getSession()
+        keyboardHandler = 
+            name: 'highlightBrackets'
+            bindKey: {win: 'Ctrl-Shift-9', mac: 'Command-Shift-9'}
+            exec: (editor)  -> return highlightBrackets(editor)
+            readOnly: true
+
+
+        editor.commands.addCommand(keyboardHandler);
+
+        session.getSelection().on("changeCursor", () -> 
+            if (session.$bracketHighlightLeft || session.$bracketHighlightRight) 
+                session.removeMarker(session.$bracketHighlightLeft)
+                session.removeMarker(session.$bracketHighlightRight)
+                session.$bracketHighlightLeft = null
+                session.$bracketHighlightRight = null
+                if (!isOutFromAreaOfHighlight())
+                    highlightBrackets(editor)
+            return
+        )
+
+        isOutFromAreaOfHighlight =  () -> 
+            oldPosition = session.$positionOfHighlight;
+            newPosition = findSurroundingBrackets(editor)
+            return (oldPosition.equals(newPosition));
+        return
+
     exports.highlighter =
         highlightBrackets: highlightBrackets
         findSurroundingBrackets: findSurroundingBrackets
+        higlightInitBind: higlightInitBind
```

Comment:
first, there is typo here: `higl..` but anyway, just name it `init` and maybe parameterize with the bindKey argument, so that we could configure the binding externally